utils/python/tests/scripts_regression_tests

```diff
@@ -488,12 +501,12 @@ class TestCreateTestImpl(TestCreateTestCommon):
         for test_status in test_statuses:
             status, test_name = wait_for_tests.parse_test_status_file(test_status)
             if (test_name == build_fail_test):
-                self.assertEqual(status[create_test_impl.BUILD_PHASE], wait_for_tests.TEST_FAIL_STATUS)
```

Comment:
I just checked out your branch and I see BUILD_PHASE being assigned as a global on line 18 of CreateTest.py.